DateSeer/Main.cs

```diff
@@ -11,208 +11,44 @@
 
 namespace DateSeer
 {
-    public partial class Main : Form
+    public partial class Main : Form,  MainForm
     {
-       private User MainUser;
-       private User Usern;
-               
-        public Main(User MainUserer)
-        {
-            this.MainUser = MainUserer;
-            MainUser.GetUserInfoByUsername();
-            string path = GetResourcesPath();
-            path = Path.Combine(path,"Users");
-            path = path + @"\" + MainUser.username + ".txt";
-            if (File.Exists(path)) { }
-            else
-            {
-                MainUser.CreateFile(path);
-            }
-            InitializeComponent();
-            Load_User();
-        }
-
-        private string GetResourcesPath()
-        {
-            string PathR = Directory.GetParent(Directory.GetCurrentDirectory()).Parent.FullName;
-            PathR = Path.Combine(PathR, "Resources");
-            return PathR;
-        }
-
-        private void Load_User()
-        {
-            Usern = new User(null, null);
-            if (MainUser.gender == 1)
-            {
-                Usern.GetUserInfoByGender(2,MainUser.getId());
-            }
-            else
-            {
-                Usern.GetUserInfoByGender(1,MainUser.getId());
-            }
-            string name; ;
-            if (Usern.name == null)
-            {
-                string pathg = GetResourcesPath();
-                pathg = Path.Combine(pathg + @"\NoMore.png");
-                Image image = Image.FromFile(pathg);
-                UsernPic.Image = image;
-                label2.Text = "";
-
-            }
-            else
-            {
-                name = Usern.name.ToString();
-                label2.Text = name;
-                string PathR = GetResourcesPath();
-                PathR = Path.Combine(PathR, "DefaultAccountPic");
-                if (Usern.getImage() == "")
-                {
-                    if (Usern.gender == 1)
-                    {
-
-                        Image image = Image.FromFile(PathR + @"\male.png");
-                        UsernPic.Image = image;
 
-                    }
-                    else
-                    {
-                        Image image = Image.FromFile(PathR + @"\female.jpg");
-                        UsernPic.Image = image;
+       private Action LikeButtomPressed_ = delegate { };

```

Comment:
If you want to signify that it is private variable, use underscore in the beginning of the name, like `_variable`; Also, the name should begin with lowercase letter